core/controllers/admin_test.py

```diff
@@ -106,6 +108,53 @@ def test_change_about_page_config_property(self):
         self.assertIn(new_config_value, response.body)
 
 
+    def test_add_notification_email_for_failed_tasks(self):
+        """ Test if admin is able to correctly change notification emails
+        for failing tasks
+        """
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+        response = self.testapp.get('/admin')
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        response_dict = self.get_json('/adminhandler')
+        response_config_properties = response_dict['config_properties']
+        self.assertDictContainsSubset({
+            'value': ['moderator@example.com'],
+        }, response_config_properties[email_manager.NOTIFICATION_EMAILS_FOR_FAILED_TASKS.name])
+
+        # check for some valid emails
+        for emails in [
+            [u'admin@oppia.com'],
+            [u'admin{}@oppia.com'.format(i) for i in xrange(
+                0, email_manager.MAX_NOTIFICATION_EMAILS)]
+        ]:
+            payload = {
+                'action': 'save_config_properties',
+                'new_config_property_values': {
+                    email_manager.NOTIFICATION_EMAILS_FOR_FAILED_TASKS.name:
+                        emails
+                }
+            }
+            self.post_json('/adminhandler', payload, csrf_token)
+
+        # check against some invalid emails
+        for emails in [
+            [u'adminoppia'],
+            [u'admin{}@oppia.com'.format(i) for i in xrange(
+                0, email_manager.MAX_NOTIFICATION_EMAILS + 1)]
+        ]:
+            payload = {
+                'action': 'save_config_properties',
+                'new_config_property_values': {
+                    email_manager.NOTIFICATION_EMAILS_FOR_FAILED_TASKS.name:
+                        emails
+                }
+            }
+            with self.assertRaises(webtest.app.AppError):
```

Comment:
Why `webtest` error? Check other tests for exception testing. Don't import webtest here directly.